File: W2_task1.js
//  要求一: 函式與流程控制

function calculate(min, max, step) {
  let total = 0
  for (let i = min; i <= max; i += step) {
    // console.log(i);
    total += i;
  }
  console.log(total);
}

calculate(1, 3, 1);
calculate(4, 8, 2);
calculate(-1, 2, 2);

File: W2_task1.py
# 要求一: 函式與流程控制

# 創建calculate函式中的list
# 創建函式、賦值(NewList的總和函式)
# 印出NewList所有數字加總

def calculate(min, max, step):
    NewList = range(min, max+1, step)
    # print(list(NewList))
    NewListSum = sum(NewList)
    print(NewListSum)


calculate(1, 3, 1)
calculate(4, 8, 2)
calculate(-1, 2, 2)
